packages/config/types/router.d.ts

```diff
@@ -9,6 +9,8 @@ import { RouterOptions, RouteConfig } from 'vue-router'
 
 export interface NuxtRouteConfig extends RouteConfig {
   chunkNames: { [key: string]: string }
+  component: RouteConfig['component'] | string
```

Comment:
```suggestion
  component?: RouteConfig['component'] | string
```

I just checked https://github.com/vuejs/vue-router/blob/dev/types/router.d.ts#L77, the property should be optional.

Reply:
Sorry for that 🤦‍♂️